cypress/integration/ProcessorTest.ts

```diff
@@ -18,7 +15,14 @@ onlyOn(isEnvironmentType(EnvType.Mocked), () => {
 
       it("Sink processor", () => {
         const processorName: string = "Sink processor";
-        const action = ["Slack", "dev-action", "https://test.app.com/item"];
+        const action = {
```

Comment:
what about having constants used as key in this `action` object?
example:
```
var CONSTANT_A = 0, CONSTANT_B = 1
var state_machine = {
    [CONSTANT_A]: function () {
        return 'a'
    },
    [CONSTANT_B]: function () {
        return 'b'
    }
};
```

in this way you can use them also in lines 72, 84, 91

Reply:
Square brackets are used inside the object definition to have variables/context inside the object's key (https://stackoverflow.com/a/68796045).
Once you have defined a constant for each string, you can use the constant directly in the mentioned rows.

Reply:
@jomarko in case that you want to use some constants for Action/Source/Filters labels can you think about importing them from Utils? My future thoughts: it would be nice to import and read these objects: https://github.com/jomarko/sandbox-ui/blob/PR-MGDOBR-1148/mocked-api/schemasData.ts#L240-L290